.github/workflows/ci.yml

```diff
@@ -62,7 +55,10 @@ jobs:
         fi
       env:
         CC_TEST_REPORTER_ID: f507ecc7b3090a909467a65b2b78de334b6e819b99b2a54c5c333e73e2c1ffbb
-
+    - name: "Add environment variables"
+      run: |
+        echo "SEARCH_URL='https://search-testy-search-testy-u2vq42wckv4epdwlul2nthzvsi.us-east-1.es.amazonaws.com'" >> sites/marble/.env.production
```

Comment:
This is not needed for indexing, but it is needed to build pages. It can be any value, but it cannot be null.